style.css

```diff
@@ -0,0 +1,100 @@
+body {
+    font-family: 'Comfortaa', cursive;
```

Comment:
The indentation is not quite right here... the line needs to move a bit to the left!